readme.md

```diff
@@ -145,16 +145,24 @@ warp deploy_account
 ### CLI Commands
 
 <hr> 
-To transpile a contract:
+To transpile a Solidity contract:
 
 ```bash
 warp transpile <path to Solidity contract>
 ```
 
-To deploy a Cairo contract:
+To declare a StarkNet contract:
 
 ```bash
-warp deploy <path to Cairo contract>
+warp declare <path to StarkNet contract>
```

Comment:
Sorry to be annoying but can you put the warp deploy_account and stuff as well
We should have intstructions that if a user follows them verbatim they should be able to deploy

Reply:
It is mentioned above. Can we make another PR for ReadME changes?